cdap-api/src/main/java/co/cask/cdap/api/workflow/WorkflowActionNode.java

```diff
@@ -28,13 +29,21 @@
   private final WorkflowActionSpecification actionSpecification;
 
   public WorkflowActionNode(String nodeId, ScheduleProgramInfo program) {
```

Comment:
Has @awholegunch reviewed the Javadocs in this package? I think they need an overhaul